splits/README.md

```diff
@@ -0,0 +1,23 @@
+# Evaluation Setup of Natural Instruction V2
+
```

Comment:
Suggested addition: 

"Here we describe the evaluation setup used in [this paper](https://arxiv.org/abs/2204.07705) which can be used for reproducing our experiments or extending them."